python/semantic_kernel/connectors/ai/ollama/services/ollama_text_completion.py

```diff
@@ -37,31 +37,31 @@ class OllamaTextCompletion(TextCompletionClientBase, AIServiceClientBase):
     async def complete(
         self,
         prompt: str,
-        request_settings: OllamaTextRequestSettings,
+        prompt_execution_settings: OllamaTextPromptExecutionSettings,
         **kwargs,
     ) -> Union[str, List[str]]:
         """
         This is the method that is called from the kernel to get a response from a text-optimized LLM.
 
         Arguments:
             prompt {str} -- The prompt to send to the LLM.
-            settings {AIRequestSettings} -- Settings for the request.
+            settings {AIPromptExecutionSettings} -- Settings for the request.
```

Comment:
My last comment, let's do a global search and replace for this -> PromptExecutionSettings. :)

Reply:
Fixed